Analysis/include/EventInfo.h

```diff
@@ -274,4 +292,14 @@ class EventInfoBase {
 
 };
 
+///not needed right??
+boost::optional<EventInfoBase> CreateEventInfo(const ntuple::Event& _event, size_t _selected_hh_index, Period _period,
+                    JetOrdering _jet_ordering, const SummaryInfo* _summaryInfo); //to be removed
+
+boost::optional<EventInfoBase> CreateEventInfo(const ntuple::Event& _event, TauIdDiscriminator _discr, Period _period,
```

Comment:
It is enough to put default values for the params, not need to declare 4 functions.